net-core/Ical.Net/Evaluation/RecurrencePatternEvaluator.cs

```diff
@@ -259,11 +259,21 @@ private void EnforceEvaluationRestrictions(RecurrencePattern pattern)
                     break;
                 }
 
+                if (pattern.Until != DateTime.MinValue && candidate != DateTime.MinValue && candidate > pattern.Until)
```

Comment:
This condition is exact duplicate of the above statement, therefore it should be removed.

Reply:
Good catch.  This was an accident.  Removed the duplication.  @pomoika